nepalingo-web/src/App.tsx

```diff
@@ -11,11 +11,11 @@ import FlashcardPage from "./pages/FlashcardPage"
 import { useAuth } from "./components/userAuth/AuthContext";
 import ReactGA from "react-ga4";
 
-const App: React.FC = () => {
-  const TrackingID = import.meta.env.VITE_GOOGLE_ANALYTICS_TRACKING_ID;
-  ReactGA.initialize(TrackingID);
-  const { user } = useAuth();
+const TrackingID = import.meta.env.VITE_GOOGLE_ANALYTICS_TRACKING_ID;
+ReactGA.initialize(TrackingID);
+const { user } = useAuth();
```

Comment:
Not sure why these got moved out of the function, but they should go inside of `const App: React.FC`
Otherwise, right now I get an error complaining that `useAuth` hook can't be called outside of a react functional component:
![CleanShot 2024-07-17 at 22 21 05@2x](https://github.com/user-attachments/assets/90549ad1-f923-4f66-a3ac-437b84e315f9)

You can see this error yourself if you run `pnpm run dev`
Then in your browser go to `localhost:5174`, then right click > Inspect > Console